core/src/components/item/test/legacy/disabled/index.html

```diff
@@ -138,8 +133,7 @@
 
           <ion-item>
             <ion-checkbox slot="start" legacy="true"></ion-checkbox>
-            <ion-label>Checkbox + Range</ion-label>
-            <ion-range disabled value="45"></ion-range>
+            <ion-label>Checkbox</ion-label>
```

Comment:
Same here - this was testing multiple inputs but won't test that anymore, we should remove entirely